lib/browser/results.js

```diff
@@ -40,6 +41,11 @@ createMethods(Results.prototype, objectTypes.RESULTS, [
     'removeAllListeners',
 ]);
 
+// Mutating methods:
+createMethods(Results.prototype, objectTypes.RESULTS, [
+    'update',
```

Comment:
Am I correct in assuming that this should be in the 'mutating' methods section?

Reply:
yes